backend/handlers/event.go

```diff
@@ -43,9 +45,104 @@ func (h *EventHandler) GetMany(ctx *fiber.Ctx) error {
 }
 
 func (h *EventHandler) GetOne(ctx *fiber.Ctx) error {
-	return nil
+	eventId := ctx.Params("eventId")
+
+	context, cancel := context.WithTimeout(context.Background(), time.Duration(5*time.Second))
+	defer cancel()
+
+	event, err := h.repository.GetOne(context, eventId)
+
+	if err != nil {
+		return ctx.Status(fiber.StatusBadRequest).JSON(&fiber.Map{
+			"status":  "fail/getone/statusbadrequest",
+			"message": err.Error(),
+		})
+	}
+
+	return ctx.Status(fiber.StatusOK).JSON(&fiber.Map{
+		"status":  "success",
+		"message": "",
+		"data":    event,
+	})
 }
 
 func (h *EventHandler) CreateOne(ctx *fiber.Ctx) error {
-	return nil
+	event := &models.Event{}
+
+	context, cancel := context.WithTimeout(context.Background(), time.Duration(5*time.Second))
+	defer cancel()
+
+	if err := ctx.BodyParser(event); err != nil {
+		return ctx.Status(fiber.StatusUnprocessableEntity).JSON(&fiber.Map{
+			"status":  "fail/createone/bodyparser",
+			"message": err.Error(),
+			"data":    nil,
+		})
+	}
+
+	event, err := h.repository.CreateOne(context, event)
+
+	if err != nil {
+		return ctx.Status(fiber.StatusBadRequest).JSON(&fiber.Map{
+			"status":  "fail/createone/statusbadrequest",
+			"message": err.Error(),
+			"data":    nil,
+		})
+	}
```

Comment:
Request: 
{
 "name": "Hoang",
 "location": "Vietnam"
}
Response:
{
    "data": null,
    "message": "ERROR: null value in column \"id\" of relation \"events\" violates not-null constraint (SQLSTATE 23502)",
    "status": "fail/createone/statusbadrequest"
}